src/components/dialogs/AdvancedFiltersDialog.js

```diff
@@ -146,24 +154,18 @@ const AdvancedFiltersDialog = ({
   }
 
   useEffect(() => {
-    if (object) {
+    if (objectType) {
       // Update the state with new parameters
       let paramsToFetchFilters = []
-      if (objectType === BENEFIT_PLAN) {
-        paramsToFetchFilters = createParams(
-          moduleName,
-          objectType,
-          object.id
-        );
-      } else {
```

Comment:
could you elaborate on why it was removed?

Reply:
Right now simply inserting id as an function argument decides if it should be in the request or not. The only job of that if statement was to add id for benefit plan.